From f81ecd6283c054f517af680ded31a3df4bcd08d9 Mon Sep 17 00:00:00 2001
From: Marian Rotariu <marian.rotariu@freescale.com>
Date: Wed, 26 Feb 2014 18:22:22 +0200
Subject: [PATCH 608/987] dpaa_eth: add print for proxy probe

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

There is no indication that the proxy interface successfully initialized the
attached MAC device. In this way, the user does not know if the device tree
is misconfigured or the MAC device successfully probed. Other ways to check
the MAC initialization exists, like MAC mem mapped registers, but a simple
message in the bootlog seems way more user-friendly.

Signed-off-by: Marian Rotariu <marian.rotariu@freescale.com>
Change-Id: I79b38719e6b49d8a4557b885a2b13854961f9b32
Reviewed-on: http://git.am.freescale.net:8181/9258
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Madalin-Cristian Bucur <madalin.bucur@freescale.com>
Reviewed-by: Jose Rivera <German.Rivera@freescale.com>
(cherry picked from commit f217d812c07a1be0563dffe40f4736442b5f99ee)
Reviewed-on: http://git.am.freescale.net:8181/9263
---
 drivers/net/ethernet/freescale/dpa/dpaa_eth_proxy.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth_proxy.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth_proxy.c
index 41478fd0..a06465a 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth_proxy.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth_proxy.c
@@ -195,6 +195,10 @@ static int dpaa_eth_proxy_probe(struct platform_device *_of_dev)
 			goto port_enable_fail;
 	}
 
+	dev_info(dev, "probed MAC device with MAC address: %02hx:%02hx:%02hx:%02hx:%02hx:%02hx\n",
+		     mac_dev->addr[0], mac_dev->addr[1], mac_dev->addr[2],
+		     mac_dev->addr[3], mac_dev->addr[4], mac_dev->addr[5]);
+
 	return 0; /* Proxy interface initialization ended */
 
 port_enable_fail:
-- 
1.9.1

